templates/cli/lib/sdks.js.twig

```diff
@@ -61,19 +61,19 @@ const sdkForProject = async () => {
     .setSelfSigned(selfSigned)
     .setLocale("en-US");
 
-  if (key) {
+  if (cookie) {
     return client
 {% if sdk.test != "true" %}
-      .setKey(key)
-      .setMode("default");
+      .setCookie(cookie)
```

Comment:
Is this a new setting?

Reply:
No i just changed the order, to give higher preference to cookies that an API key